styles.css

```diff
@@ -1,3 +1,6 @@
+/* Import Poppins font family from Google Fonts */
```

Comment:
This comment can be removed - the line below is very clear without aditional comment